.github/workflows/validate.yml

```diff
@@ -2,33 +2,31 @@ name: validate
 on:
   push:
     branches:
-      - '+([0-9])?(.{+([0-9]),x}).x'
-      - 'master'
-      - 'next'
-      - 'next-major'
-      - 'beta'
-      - 'alpha'
-      - '!all-contributors/**'
+      - "+([0-9])?(.{+([0-9]),x}).x"
+      - "master"
+      - "next"
+      - "next-major"
+      - "beta"
+      - "alpha"
+      - "!all-contributors/**"
   pull_request: {}
 jobs:
   main:
     # ignore all-contributors PRs
     if: ${{ !contains(github.head_ref, 'all-contributors') }}
     strategy:
       matrix:
-        node: [10.13, 10, 12, 14, 15]
+        node: [10.13, 10, 12, 14]
```

Comment:
I saw 15 failing in the CI in an earlier push. Is this why we're removing it from the workflow? I don't have strong feelings against it because 15 is not very common yet, but it might be worth looking into why it failed instead of disabling the run

Reply:
I found the problem as you can see in https://github.com/kentcdodds/kcd-scripts/issues/200#issuecomment-817325139

This will be resolved once we drop `gatsby@2.x` and update to `gatsby@3.x`